packages/common/src/util/base64.js

```diff
@@ -22,5 +30,13 @@ export const encode = data => Buffer.from(data, 'utf-8').toString('base64');
  * const decoded = decode('SGVsbG8gV29ybGQ=');
  * console.log(decoded); // Output: Hello World
  */
-export const decode = base64Data =>
-  Buffer.from(base64Data, 'base64').toString('utf-8');
+export const decode = base64Data =>{
+  let decodedObject = Buffer.from(base64Data, 'base64').toString('utf-8');
+
+  if(['[', '{'].includes(decodedObject[0])){
+    decodedObject = JSON.parse(decodedObject);
```

Comment:
As with `encode`, we should catch any errors here, log a warning, and return the plain string form. Just in case.